billing.go

```diff
@@ -109,48 +108,48 @@ type Payer struct {
 
 	// User payer only
 	UserID    *string `json:"user_id"`
-	FirstName *string `json:"first_name"`
-	LastName  *string `json:"last_name"`
-	Email     *string `json:"email"`
+	FirstName *string `json:"first_name,omitempty"`
+	LastName  *string `json:"last_name,omitempty"`
+	Email     *string `json:"email,omitempty"`
 
 	// Org payer only
 	OrganizationID   *string `json:"organization_id"`
 	OrganizationName *string `json:"organization_name"`
 
 	// Used for both org and user payers
-	ImageURL string `json:"image_url"`
+	ImageURL string `json:"image_url,omitempty"`
```

Comment:
The omitempty doesn't really do anything for this, but I think making it a pointer would be idiomatic. It would allow people to detect when the ImageURL wasn't set.